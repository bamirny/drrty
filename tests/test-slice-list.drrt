def int main(){
	list[int] score;
	score = [12,31,40,90,111];
	print(score[1]);
	
	score = slice(score,0,3);
	print(score[0]);
	

	return 0;

}


