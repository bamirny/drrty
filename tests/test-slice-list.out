31<br>
12<br>
